Ghost_in_the_Shell/client_folder_react_app/ethisim/src/components/scenario_components/InformationItem.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import SunEditor from "suneditor-react";
+import 'suneditor/dist/css/suneditor.min.css';
+import Button from '@material-ui/core/Button';
+
+export default function InformationItem() {
+  return (
+    <div>
+      <SunEditor
```

Comment:
You should look into where @Enoch-Hsiao implemented this in his PR, you can just copy paste a few things. Specifically he got it more intractable on mobile. Maybe that should be be put in its own file with the settings and then imported across the app so you dont have to keep copy pasting!